lib/httpAuthMiddleware.js

```diff
@@ -0,0 +1,66 @@
+const { Passport } = require('passport')
+const { Strategy } = require('./strategy')
```

Comment:
Is there a file called strategy missing in this PR?

Reply:
No - the file already exists. We reuse the strategy from the adminAuth side of things